DataHub/Operation/01_ManageETLMappings.md

```diff
@@ -0,0 +1,316 @@
+[!!Omni-Channel](../../Channels/Overview/01_General.md)
+[!!User Interface ETL](../UserInterface/02d_ETL.md)
+[!!Manage an attribute set](../Integration/02_ManageAttributeSets.md)
+[!!Manage the ETL processes](./02_ManageETLProcesses.md)
+[!!Data type list](../UserInterface/04_DataTypeList.md)
+[!!ETL extensions list](../UserInterface/03_ETLExtensions.md)
+
+
+# Manage the ETL mappings
+
+An ETL attribute set mapping is used to extract the data from a source attribute set, transform it according to a specified method defined by the extension, and load it into a destination attribute set.
+
+You can create mappings, edit existing mappings and rerun mappings.
+
+
+## Create an ETL attribute set mapping
+
+Create an ETL attribute set mapping to map the attributes of the source attribute set to the attributes of the destination attribute set.
+The mapping allows to convert the attributes from the source set to fit the attributes required by the destination set.
+
+#### Prerequisites
+
+At least two attribute sets have been created, see [Create an attribute set](./02_ManageAttributeSets.md#Create-an-attribute-set).
+
+#### Procedure
+
+*DataHub > Settings > Tab ETL*
+
+![Attribute set mappings](../../Assets/Screenshots/DataHub/Settings/ETL/AttributeSetMappings.png "[Attribute set mappings]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+  The *Create ETL attribute set mapping* window is displayed.
+
+  ![Create ETL attribute set mapping](../../Assets/Screenshots//DataHub/Settings/ETL/CreateETLAttributeSetMapping.png "[Create ETL attribute set mapping]")
+
+2. Select a source attribute set in the *Source attribute set* drop-down list. All active attribute sets are displayed in the drop-down list.
+
+3. Select a destination attribute set in the *Destination attribute set* drop-down list. All active attribute sets are displayed in the drop-down list.   
+  If any attribute set mapping with the selected destination attribute set already exists, the *Copy ETL attribute set mapping* drop-down list is unlocked.
+
+4. Select a language in the *Language* drop-down list. All active languages are displayed in the drop-down list.
+
+  > [Info] The selected language defines the language from which the value of a multilingual attribute is taken when mapping it to a single language attribute.
```

Comment:
multilingual vs. multi-language

In DataHub und in PIM -> Sind sie Synonyme oder nicht ganz? Vereinheitlichen?